### PR TITLE
Docusaurus2

### PR DESCRIPTION
**Pull request by kingsleyzissou**
- Migrated docs to docusarus 2.
- New layout and design of the offix website

##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

File: website/src/components/Introduction/index.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export function Introduction() {
+  return (
+    <Flex background="#009CC6" minHeight="100vh">
+      <Container>
+        <Row>
+          <Flex height="100%" width="50%" color="#fff">
+            <div> 
+              <Title>Offline First!</Title>
+              <Paragraph>Offix allows you to execute your GraphQL mutations and queries while your application is offline (or while the server is unreachable). Offline Mutations are scheduled and persisted (across application restarts) and are replayed when server becomes available again.</Paragraph>
```
Comment (darahayes) [inline website/src/components/Introduction/index.jsx:13]:
```suggestion
              <Paragraph>Offix allows you to execute your GraphQL mutations and queries while your application is offline, or while the server is unreachable. Offline Mutations are scheduled and persisted across application restarts, and are replayed when server becomes available again.</Paragraph>
```

**Comment by wtrocki** [comment]
Got it running locally. Works with the mobile and web! 
Amazing job!

**Comment by wtrocki** [comment]
@kingsleyzissou For missing sections we can put:

- some code snippets 
- Photo/GIF of starter app with offix logo embeeded.

**Comment by kingsleyzissou** [comment]
Thanks for the feedback guys, I've fixed the intro section and added the scrollmagic dependency. Will sort out the highlight section this morning.

**Comment by kingsleyzissou** [comment]
@darahayes @wtrocki I just went for a simple solution in the end for the highlights section.

I replaced the video with a gif demo of the showcase and used the orange background

**Comment by wtrocki** [comment]
I think we should limit the number of versions we have build

**Comment by wtrocki** [comment]
I also see a lots of filles that are not used like README.md etc.

Also see #474 - I made changes on the old version - was expecting this to be merged it we published

**Comment by wtrocki** [comment]
Edit url goes to graphback:

https://github.com/aerogear/graphback/edit/master/website/

**Comment by wtrocki** [comment]
docs/CNAME nojeckyl and 404 are unused

**Comment by wtrocki** [comment]
Considering number of releases from now I think we will be better off to keep 2-3 releases for this

**Comment by wtrocki** [comment]
Video is no longer used

**Pull request merged**